### PR TITLE
Add SPDX header

### PR DESCRIPTION
**Pull request by AndyScherzinger**
... and an action checking upon changes

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import Tag from './Tag.vue'
 
+/**
```
Comment (nickvergessen) [inline src/main.js:3]: I guess it should be above the code

Comment (AndyScherzinger) [reply src/main.js:3]: I am not an expert on these languages, but sure I can move it up if that is fine - thanks for the feedback 👍

Comment (AndyScherzinger) [reply src/main.js:1]: I pushed https://github.com/nextcloud/files_automatedtagging/pull/912/commits/b4eaeee1b988e6910229489230a0c8f342902248 (if that is hat you suggested)

Comment (nickvergessen) [reply src/main.js:1]: yes

File: .github/workflows/reuse.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+# SPDX-FileCopyrightText: 2022 Free Software Foundation Europe e.V. <https://fsfe.org>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+name: REUSE Compliance Check
+
+on: [push, pull_request]
```
Comment (nickvergessen) [inline .github/workflows/reuse.yml:12]:
```suggestion
on: pull_request
```

Comment (AndyScherzinger) [reply .github/workflows/reuse.yml:12]: Would it still get executed when you push further commits to a branch with an open PR?

File: .reuse/dep5 [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: files_automatedtagging
+Upstream-Contact: Nextcloud <info@nextcloud.com>
+Source: https://github.com/nextcloud/files_automatedtagging
+
+Files: .github/* .editorconfig babel.config.js .php-cs-fixer.dist.php package-lock.json package.json composer.json composer.lock README.md img/app-dark.svg img/app.svg webpack.js stylelint.config.js .eslintrc.js .gitignore .jshintrc .l10nignore action/.gitignore action/package.json action/package-lock.json action/dist/index.js tests/* psalm.xml vendor-bin/*/composer.json vendor-bin/*/composer.lock .tx/config webpack.config.js
```
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply .reuse/dep5:6]: Also not sure, what would you say for the license? I can also ask @jancborchardt what the license of the icons is / should be.

File: .github/workflows/reuse.yml (hunk 2 above)
Comment (nickvergessen) [reply .github/workflows/reuse.yml:12]: yes, push is for pushing to master, etc which is not allowed anyway or no one looks at the CI result.

Comment (AndyScherzinger) [reply .github/workflows/reuse.yml:12]: 👍 fixed in the workflow repo via the SPDX PR 👍

File: .reuse/dep5 (hunk 3 above)
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply .reuse/dep5]: Replaced with an Material icon via https://github.com/nextcloud/files_automatedtagging/pull/912/commits/1ae179de6f80493a77b31a8cafa9d0b554567c8c

**Comment by AndyScherzinger** [comment]
👍

**Comment by AndyScherzinger** [comment]
@nickvergessen tests are failing but I have a hard time understanding why this would be related to changed comments.

**Comment by nickvergessen** [comment]
Someone broke the public interface I guess

File: tests/OperationTest.php [hunk 4]
```diff
@@ -1,22 +1,9 @@
 <?php
+
+declare(strict_types=1);
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply tests/OperationTest.php:3]: You can also remove it and raise an issue for me

[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply tests/OperationTest.php:3]: No hurries, Thursday (or later) sounds perfectly fine to me 🙏

**Pull request merged**